docs/blog/newsletter-april-2022.rst

```diff
@@ -0,0 +1,137 @@
+- Certifications, training, and freelance opportunities are all valid ways to create a portfolio and gain a basic level of industry knowledge.
+- Leverage your experience even if you’re coming from a completely different field: there are often relevant skills you can highlight, like other kinds of writing, or research skills.
+- Contribute and engage in open-source communities to show your skills and passion.
+- Look for internship programs, for example through `Outreachy <http://outreachy.org/>`__, to find low-barrier entry opportunities.
+- Update your resume by adding new skills, linking your portfolio and social media, and tailoring your experience to each job posting.
+- Network with tech writers and find mentors to learn about new roles, share experiences, and develop professional relationships.
+- Lastly - don’t give up! These things take time and effort, but the community is here to support you - drop by in `#career-advice <https://app.slack.com/client/T0299N2DL/C6ADX1YVA>`__ if you need a boost.
+
+------------------------------------
+Can form follow function in techdoc?
+------------------------------------
+
+What matters more for documentation: "functional quality" (helps the reader get stuff done) or "structural quality" (looks nice, follows the style guide)? A lively discussion about which of two qualities matters more for documentation unsurprisingly produced the conclusion, "Why not both?" Along the way, however, folks contributed a plethora of refining points and important additions to the initial comparison. Here's a summary:
+
+It's easier to think in terms of structural quality - grammar, word choice, appropriate headings, ease of scanning a page. It's easier to check and to test, with linters and other automation tools.
+
+But functional quality matters more. Do the docs achieve their goal of helping users get their work done? Testing this is harder - you can try by manually testing steps in docs, but that isn't a full answer.
+
+But when you dig into the details of functional characteristics, the distinction between functional and structural starts to blur. If all the information a user needs is available, but is so badly organised that the user can't accomplish their goals, then functional quality is compromised by the lack of structural quality.
+
+A refinement of this point distinguishes between the goals of the user -- does the doc help the user accomplish their goals? and the goals of the writer or the company they work for. If a doc helps the user, but fails a structural quality test, it might be costly or difficult to maintain, or might not integrate well with the rest of the docs. And these problems can become a failure of functional quality for the writer or larger doc set.
```

Comment:
```suggestion
A refinement of this point distinguishes between the goals of the user -- does the doc help the user accomplish their goals? -- and the goals of the writer or the company they work for. If a doc helps the user, but fails a structural quality test, it might be costly or difficult to maintain, or might not integrate well with the rest of the docs. And these problems can become a failure of functional quality for the writer or larger doc set.
```